modules/removeBook.js

```diff
@@ -0,0 +1,25 @@
+import storageAvailable from './storageAvailable.js';
+import updateStorage from './updateStorage.js';
+import populateBook from './populateBook.js';
+
+const removeBook = (event) => {
+  let bookList = [];
+
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    bookList = JSON.parse(localStorage.getItem('bookList'));
+  }
+
+  const bookDiv = document.getElementsByClassName('book-list')[0];
+  const objectId = event.target.id;
+
+  const newBookList = bookList.filter((book) => book.Title !== bookList[objectId].Title);
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you implement your book removal using a sort of unique id. Right now, when I add two books of the same title and remove one, both books are removed, but this should not be so.